history/views.py

```diff
@@ -93,7 +93,7 @@ def info_button(self, instance, *args, **kwargs):
             # this is much faster than the routine config_dict.
             config_dict = json.loads(instance.configuration)
             for key, value in config_dict.items():
-                text = escape(mask_uid(str(value), request.user.isGuest()))
+                text = escape(mask_uid(str(value), request.session.get("user_home_dir", "") == ""))
```

Comment:
I am not sure whether this is ok, home dirs are defined only in a ldap or kerberos scenario. We don't want ourself to restrict to those identidy providers only.